undistro/zora
# UD-1540: Increase default size of PV to 2Gi

## Description
Increase the size of the PV to 2Gi, to allow for both databases to be downloaded and persisted

## Linked Issues

## How has this been tested?
- Configure trivy-download-db to download java db on installation
- Ensure job and scan work without errors related to disk space

## Checklist
